sprokit/src/sprokit/pipeline/pipeline.cxx

```diff
@@ -2175,7 +2175,7 @@ ::check_port_frequencies() const
     process::port_frequency_t const& freq = proc_freq.second;
     process::frequency_component_t const denom = freq.denominator();
 
-    freq_gcd = boost::math::lcm(freq_gcd, denom);
+    freq_gcd = boost::integer::lcm(freq_gcd, denom);
```

Comment:
boost::xxx::lcm() is only used in one place. This could be replaced with a local implementation and this would remove one more boost dependency.

Reply:
I had the same thought this morning.  Dave is looking into upgrading Boost in Fletch and dropping support for the old Boost.  So ultimately we could accept this change and not worry about old Boost.  However, this might be a good excuse to just replace this function with a non-Boost alternative.  I haven't look to see how hard this is to reimplement, but it can't be too hard.  It's just finding the lowest common multiple of two integers.

Reply:
I was thinking about clipping some code from stack-overflow, but maybe there is a better source for the LCM algorithm.

Reply:
You could also just copy the code from Boost and greatly simplify it so it is no so generic and just works for the types that that this sprokit function cares about.